db/sql/001-create-timeEntry-and-timePeriodType-table.sql

```diff
@@ -0,0 +1,34 @@
+
+CREATE SCHEMA IF NOT EXISTS timecard;
+
+CREATE TABLE IF NOT EXISTS timecard.time_period_type (
+    id VARCHAR(36)  PRIMARY KEY,
+    tenant_id VARCHAR(36) NOT NULL,
+    name VARCHAR(50) NOT NULL
+);
+
+INSERT INTO timecard.time_period_type (id,tenant_id,"name") VALUES
+	 ('00000000-0000-0000-0000-000000000001','00000000-0000-0000-0000-000000000000','Shift'),
+	 ('00000000-0000-0000-0000-000000000002','00000000-0000-0000-0000-000000000000','Scheduled rest day'),
+	 ('00000000-0000-0000-0000-000000000003','00000000-0000-0000-0000-000000000000','Non-working day'),
+	 ('00000000-0000-0000-0000-000000000004','00000000-0000-0000-0000-000000000000','On call'),
+	 ('00000000-0000-0000-0000-000000000005','00000000-0000-0000-0000-000000000000','Absence'),
+	 ('00000000-0000-0000-0000-000000000006','00000000-0000-0000-0000-000000000000','Training'),
+	 ('00000000-0000-0000-0000-000000000007','00000000-0000-0000-0000-000000000000','Overtime');
+
+CREATE TABLE IF NOT EXISTS timecard.time_entry (
+    id VARCHAR(36)  PRIMARY KEY,
+    tenant_id VARCHAR(36) NOT NULL,
+    version INT DEFAULT NULL,
+    owner_id INT NOT null,
+    time_period_type_id VARCHAR(36) NOT NULL,
+    shift_type VARCHAR(50) DEFAULT NULL,
+    actual_start_time TIMESTAMP NOT NULL,
+    actual_end_time TIMESTAMP DEFAULT NULL,
+    planned_start_time TIMESTAMP DEFAULT NULL,
+    planned_end_time TIMESTAMP DEFAULT NULL,
+    created_at TIMESTAMP NOT NULL,
+    updated_at TIMESTAMP NOT NULL,
+    deleted BOOLEAN DEFAULT NULL,
+    FOREIGN KEY (time_period_type_id) REFERENCES timecard.time_period_type (id)
```

Comment:
We discussed this morning not adding database fields yet for stuff we haven’t built yet. I really rate this idea. Would this in mind, would people be happy if we were to remove the following fields?

- `version`
- `planned_start_time`
- `planned_end_time`
- `deleted`

```suggestion
    id VARCHAR(36)  PRIMARY KEY,
    tenant_id VARCHAR(36) NOT NULL,
    owner_id INT NOT null,
    time_period_type_id VARCHAR(36) NOT NULL,
    shift_type VARCHAR(50) DEFAULT NULL,
    actual_start_time TIMESTAMP NOT NULL,
    actual_end_time TIMESTAMP DEFAULT NULL,
    created_at TIMESTAMP NOT NULL,
    updated_at TIMESTAMP NOT NULL,
    FOREIGN KEY (time_period_type_id) REFERENCES timecard.time_period_type (id)
```